### PR TITLE
Fix: Human Eval for new endpoints

### PR DESCRIPTION
**Pull request by error9098x**
- This is a fix for new endpoints for Human Eval
- Now, we can choose from all the available endpoints for the Human Eval

- [x] ruff passed
- [x] signed off

### CODE REVIEW

File: evaluation/human_evaluation/main.py [hunk 1]
```diff
@@ -14,16 +14,25 @@
 def main() -> None:
     load_dotenv()
 
-    GOOGLE_SHEET_ID = os.getenv("GOOGLE_SHEET_ID")
-    GOOGLE_FORM_ID = os.getenv("GOOGLE_FORM_ID")
+    google_sheet_id = os.getenv("GOOGLE_SHEET_ID")
```
Comment (luarss) [inline evaluation/human_evaluation/main.py:17]: Should it error out if the env var is not specified?

File: evaluation/human_evaluation/utils/api.py [hunk 2]
```diff
@@ -2,11 +2,20 @@
 import os
 from typing import Any
 
-API_URL = os.getenv("CHAT_ENDPOINT", "http://localhost:8000/chatApp")
+API_BASE_URL = os.getenv("CHAT_ENDPOINT", "http://localhost:8000")
 HEADERS = {"accept": "application/json", "Content-Type": "application/json"}
 
+def fetch_endpoints() -> list[str]:
```
Comment (luarss) [inline evaluation/human_evaluation/utils/api.py:8]: Are you returning a json? or a list[str]? Also note that your output type needs to handle all cases.

Comment (error9098x) [reply evaluation/human_evaluation/utils/api.py:8]: It's returning a list[str] but in case it's not able fetch it then it will just return empty list [].

Comment (luarss) [reply evaluation/human_evaluation/utils/api.py:8]: Yes but your output type will not pass mypy checks. Try running `python -m mypy --strict evaluation` to make your code mypy compliant.

**Review by luarss — APPROVED** [review]

**Pull request merged**